jsonpb/jsonpb.go

```diff
@@ -904,7 +904,7 @@ func (u *Unmarshaler) unmarshalValue(target reflect.Value, inputValue json.RawMe
 
 	// Handle nested messages.
 	if targetType.Kind() == reflect.Struct {
-		if target.CanAddr() {
+		if prop != nil && len(prop.CustomType) > 0 && target.CanAddr() { // jdef
```

Comment:
What is the comment for?
If you want you can add yourself to the CONTRIBUTORS or AUTHORS as appropriate.